src/js/Array.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-extend-native */
+
+Array.prototype.min = function () {
+  let min = this[0];
+  for (let i = 1; i < this.length; i += 1) {
+    if (min > this[i]) {
+      min = this[i];
+    }
+  }
+  return min;
+};
+Array.prototype.max = function () {
+  let max = this[0];
+  for (let i = 1; i < this.length; i += 1) {
+    if (max < this[i]) {
+      max = this[i];
+    }
+  }
+  return max;
+};
+Array.prototype.avg = function () {
+  let sum = 0;
+  for (let index = 0; index < this.length; index += 1) {
+    sum += this[index];
+  }
+  return sum / this.length;
+};
```

Comment:
[optional]
You were supposed to create a function which takes an array as an input and returns an object like the following ->
Suppose the method name as arrayAnalysis and an array [1, 2, 3, 4, 5, 6]. So if you call arrayAnalysis([1, 2, 3, 4, 5, 6]) should return.
```js
{
    average: 3.50,
    min: 1,
    max: 6,
    length: 6,
}
```
I am really sorry this was not mentioned earlier.